### ISSUE TITLE
Filter Options Cutoff On Lower Filters

### ISSUE THREAD

**Issue by gmoler** [autobrr/autobrr#1597]
**Version**
v1.43.0

**Describe the bug**
When attempting to access the '3 dot' options of a given filter, if they are on the lower part of the screen, the options are cut off. Once the options are visual, there should be a way to either scroll down, or render the page to add the options menu length to the bottom so you can see the last filters option menu

**To Reproduce**
1. Go to 'filters'
2. Have a filter that is at the bottom of the screen
3. Click on '...'
4. Notice you cannot see the options

**Expected behavior**
I expect that the page is rendered long enough to scroll all the way down, so when you click the bottom most filter, you can see the options

**Screenshots**
![image](https://github.com/user-attachments/assets/9dfbddd8-f33f-43e7-bbda-07d5e5e5fb12)

**Desktop (please complete the following information):**
 - OS: win11
 - Browser: reproducible on edge and firefox
 - Version: edge 126.0.2592.102, firefox 128

**Smartphone (please complete the following information):**
n/a

**Additional context**
n/a

**Comment by zze0s** [comment]
Thanks for the report! That's annoying, we'll get this fixed.
